<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Functions</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_compare">
        <property name="text">
         <string>Char Compare</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cleanup">
        <property name="text">
         <string>Clean up</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>TextOutput</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>AutoDisplay</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="chk_digit">
        <property name="text">
         <string>isDigit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_letter">
        <property name="text">
         <string>isLetter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_lon">
        <property name="text">
         <string>isLetterOrNum</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_upper">
        <property name="text">
         <string>isUpper</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_lower">
        <property name="text">
         <string>isLower</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_mark">
        <property name="text">
         <string>isMark</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_space">
        <property name="text">
         <string>isSpace</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_symbol">
        <property name="text">
         <string>isSymbol</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_punct">
        <property name="text">
         <string>isPunct</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>TextInput</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>String</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_str"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_unicode">
          <property name="text">
           <string>Uni to each char</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Char</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_char"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_properties">
          <property name="text">
           <string>properties</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
